README.md

```diff
@@ -17,18 +17,24 @@ It's a challenge about create a whole project, from beginning to end, should be
 See [issue #1](https://github.com/paulodiovani/hacktoberrank/issues/1) for more details.
 
 ## Project setup
+
+### Redis server is needed
 ```
-npm install
+sudo apt install redis-server
 ```
```

Comment:
I would rather add a _dependencies_ section and just list _nodejs >= 10.16.0_ and _redis_.

Not everyone uses debian/ubuntu. :P